### ISSUE TITLE
memory optimization -question

### ISSUE THREAD

**Issue by alevillard** [vlfeat/matconvnet#580]
just a question, when I use finetune some layers, leaving the upper layers with pretrained parameters and learning rate=0. It will be more faster in computation(for less learning parameters) and less memory ram space need, or MatConvNet always make the same amount of computations independently if learning rate is =0 for some layers? Thank you
